src/delimiter.js

```diff
@@ -440,34 +455,39 @@ const makeSqrtImage = function(
 
     if (delim.type === "small") {
         // Get an SVG that is derived from glyph U+221A in font KaTeX-Main.
-        viewBoxHeight = 1000 + vbPad;  // 1000 unit glyph height.
+        // 1000 unit normal glyph height.
+        viewBoxHeight = 1000 + 1000 * extraViniculum + vbPad;
         if (height < 1.0) {
             sizeMultiplier = 1.0;   // mimic a \textfont radical
         } else if (height < 1.4) {
             sizeMultiplier = 0.7;   // mimic a \scriptfont radical
         }
-        spanHeight = (1.0 + emPad) / sizeMultiplier;
-        texHeight = 1.00 / sizeMultiplier;
-        span = sqrtSvg("sqrtMain", spanHeight, viewBoxHeight, options);
+        spanHeight = (1.0 + extraViniculum + emPad) / sizeMultiplier;
```

Comment:
> Should we be adding `2 * extraViniculum` here or is the value of `emPad` just a coincidence?

It's just a coincidence. `emPad` is our estimate of how far is enough to prevent a user agent from cropping the image.